src/analytics-manager.js

```diff
@@ -126,11 +126,17 @@ var AnalyticsManager = {
     return pathInfo;
   },
 
-  trackPageView: function(freshPage, optionalTitle) {
+  trackPageView: function(freshPage, optionalTitle, optionalGaPrefix) {
     var path = this.pathInfo();
+    if (optionalGaPrefix) {
+      var gaPrefix = optionalGaPrefix;
+    }
+    else {
+      var gaPrefix = 'adTracker';
+    }
     if (this.trackedPaths.indexOf(path) < 0) {
       ga('send', 'pageview', path);
```

Comment:
This is the line we would be prefixing.  The `adTracker` must stay in hardcoded.